phalcon/mvc/view/engine/volt/compiler.zep

```diff
@@ -2210,8 +2219,23 @@ class Compiler implements InjectionAwareInterface
 				}
 				let this->_autoescape = autoescape;
 			}
+
+            /**
+             * Configures the compiler to register PHP Functions automatically.
+             */
+            if isset options["includePhpFunctions"] {
+                let includePhpFunctions = options["includePhpFunctions"];
```

Comment:
Well technically I wouldn't really need to use function_exists. Just loop through the list of functions. If not set from the option, set default to be actual functions. I see no reason as why not...